### PR TITLE
Changed all uppercase records in ratu tables to lowercase

### PR DESCRIPTION
**Pull request by Anntroy**
Changed all uppercase records in ratu tables to lowercase.

### CODE REVIEW

File: data_ocean/services/ratu_service.py [hunk 1]
```diff
@@ -64,9 +65,16 @@ def save_to_region_table(self, record):
         return region
     
     #writing entry to district table    
-    def save_to_district_table(self, record, region):
+    def save_to_district_table(self, record, region):
```
Comment (criticus) [inline data_ocean/services/ratu_service.py:68]: please remove extra spaces

File: data_ocean/services/ratu_service.py [hunk 2]
```diff
@@ -83,9 +91,9 @@ def save_to_district_table(self, record, region):
         return district
 
     #writing entry to city table    
-    def save_to_city_table(self, record, region, district):
+    def save_to_city_table(self, record, region, district):
```
Comment (criticus) [inline data_ocean/services/ratu_service.py:94]: remove extra spaces

File: data_ocean/services/ratu_service.py [hunk 3]
```diff
@@ -127,14 +135,14 @@ def save_to_citydistrict_table(self, record, region, district, city):
         return citydistrict
     
     #writing entry to street table
-    def save_to_street_table(self, record, region, district, city, citydistrict):    
+    def save_to_street_table(self, record, region, district, city, citydistrict):
```
Comment (criticus) [inline data_ocean/services/ratu_service.py:138]: extra spaces

File: data_ocean/services/ratu_service.py [hunk 4]
```diff
@@ -64,9 +65,16 @@ def save_to_region_table(self, record):
         return region
     
     #writing entry to district table    
-    def save_to_district_table(self, record, region):
+    def save_to_district_table(self, record, region):              
         if record['REGION_NAME']:
-            district_name=record['REGION_NAME']
+            position1_num = record['REGION_NAME'].find('р-н')
```
Comment (criticus) [inline data_ocean/services/ratu_service.py:70]: consider incapsulating it in a function

File: data_ocean/services/ratu_service.py (hunk 2 above)
Comment (Anntroy) [reply data_ocean/services/ratu_service.py:94]: ok

File: data_ocean/services/ratu_service.py (hunk 1 above)
Comment (Anntroy) [reply data_ocean/services/ratu_service.py:68]: ok

File: data_ocean/services/ratu_service.py (hunk 4 above)
Comment (Anntroy) [reply data_ocean/services/ratu_service.py]: done

**Pull request closed without merging**